### PR TITLE
update momentum to use Berkshelf…

### PR DESCRIPTION
**Pull request by izakp**
…and load rake tasks from non-Rails projects

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ SSH to an OpsWorks instance. If the `layer_or_instance` argument is a layer, an
 
 ### ow:execute_recipe[to,layer,recipe,aws_id,aws_secret]
 
-Execute a Chec recipe on the given layer in the given stack. By default, will execute recipes on all running instances of the _rails-app_ layer, or the list configured in `app_layers`. E.g.:
+Execute a Checf recipe on the given layer in the given stack. By default, will execute recipes on all running instances of the _rails-app_ layer, or the list configured in `app_layers`. E.g.:
```
Comment (joeyAghion) [inline README.md]: new typo

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+0.1.0 (2016-01-18)
+------
+
+* Update momentum to use Berkshelf and load rake tasks from non-Rails projects ([#19](https://github.com/artsy/momentum/pull/19))
```
Comment (joeyAghion) [inline CHANGELOG.md]: Could you note the breaking changes here (Berksfile dependency, `cookbooks_install_path`)?

Also, does this assume projects use the latest Chef/Opsworks combo? If so, we should mention that here and in the README.

Comment (izakp) [inline CHANGELOG.md]: Ok will add the notes.  The Chef/Opsworks gem versions stay as they were.

**Comment by izakp** [comment]
Actually @joeyAghion the move from librarian-chef to berkshelf means we don't depend on the chef get at all any longer.  Only the aws-sdk gem remains.

**Comment by joeyAghion** [comment]
Very nice. So, as I understand it, stacks can upgrade to this momentum version without needing to switch their stack to the Chef 12 style.

**Comment by izakp** [comment]
Yes berks will still create an archive from all dependencies as noted in the Berksfile... To upgrade a stack, the Cheffile to Berksfile will have to be re-written and the only stack configuration change should be `app-name-cookbooks.zip` -> `app-name-cookbooks.tar.gz`

**Pull request merged**